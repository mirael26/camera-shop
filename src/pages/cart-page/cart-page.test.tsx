import { renderWithReduxAndRouter } from '../../test/helpers/render-with-redux-and-router';
import CartPage from './cart-page';

jest.mock('../../components/header/header', () => 'Header');
jest.mock('../../components/breadcrumbs/breadcrumbs', () => 'Breadcrumbs');
jest.mock('../../components/footer/footer', () => 'Footer');
jest.mock('../../components/modal/modal', () => 'Modal');
jest.mock('../../components/cart/cart', () => 'Cart');

test('CartPage renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<CartPage/>);
  expect(asFragment()).toMatchSnapshot();
});
