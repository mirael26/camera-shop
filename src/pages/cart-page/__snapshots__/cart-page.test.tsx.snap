// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartPage renders correctly 1`] = `
<DocumentFragment>
  <div
    class="wrapper"
  >
    <header />
    <modal />
    <main>
      <div
        class="page-content"
      >
        <breadcrumbs />
        <cart />
      </div>
    </main>
    <footer />
  </div>
</DocumentFragment>
`;
